website/docs/r/policy_site.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "Beta"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_site"
+description: A resource to configure Policy Site.
+---
+
+# nsxt_policy_site
+
+This resource provides a method for the management of Policy Site.
+
+This resource is applicable to NSX Global Manager.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_site" "test" {
+  display_name = "test"
+  description  = "Terraform provisioned Site"
+  site_connection_info {
+    fqdn       = "192.168.230.230"
+    username   = "admin"
+    password   = "somepasswd"
+    thumbprint = "207d65dcb6f17aa5a1ef2365ee6ae0b396867baa92464e5f8a46f6853708b9ef"
+  }
+  site_type = "ONPREM_LM"
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) Display name of the resource.
+* `description` - (Optional) Description of the resource.
+* `tag` - (Optional) A list of scope + tag pairs to associate with this resource.
+* `nsx_id` - (Optional) The NSX ID of this resource. If set, this ID will be used to create the resource.
+* `fail_if_rtep_misconfigured` - (Optional) Fail onboarding if RTEPs misconfigured. Default is true.
+* `fail_if_rtt_exceeded` - (Optional) Fail onboarding if maximum RTT exceeded. Default is true.
+* `maximum_rtt` - (Optional) Maximum acceptable packet round trip time (RTT). Default is 250.
```

Comment:
The NSX API guide does not mention it, but I think we should specify the unit of measure.
msecs, I believe.

Reply:
IDK which are the units as I couldn't find that attribute in the UI